pkg/runtime/validate.go

```diff
@@ -103,6 +103,10 @@ func (e *Executor) ValidateInputs() error {
 	}
 	zap.S().Debugf("using policy path %v", e.policyPath)
 
+	if len(e.categories) > 0 && !utils.ValidateCategoryInput(e.categories) {
+		return errSeverityNotSupported
+	}
+	zap.S().Debugf("using categories %v", e.categories)
 	if len(e.severity) > 0 && !utils.ValidateSeverityInput(e.severity) {
```

Comment:
A blank line before this new if clause would make the code look a bit more near.

Reply:
sure ✌️